pkg/services/ngalert/state/manager_private_test.go

```diff
@@ -3689,14 +3689,17 @@ func TestProcessEvalResults_StateTransitions(t *testing.T) {
 				},
 			},
 			{
-				desc: "t1[QueryError] t2[{}:normal] at t2",
+				desc: "t1[QueryError] t2[{}:normal] at t2,t3",
```

Comment:
please change description to "t1[QueryError] t2[{}:normal] t3[{}:normal] at t2,t3",

Reply:
updated!